www-src/app/replicator/replicator_config.coffee

```diff
@@ -45,6 +43,21 @@ module.exports = class ReplicatorConfig extends Backbone.Model
             deviceName: changes.deviceName
 
         @set changes
+
+        return needInit
+
+    ###*
+     *
+     * @param changes [optional] object with attributes to changes
+     * @param callback
+    ###
+    save: (changes, callback) ->
+        log.info "save changes"
+        if arguments.length is 2
+            @set changes if changes?
+        else if arguments.length is 1
+            callback = changes
```

Comment:
Just making changes arg optionnal : if there is only one argument, then it is a callback.
I may write callback = arguments[0] , it may be less tricky to figure out